.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Release pipeline
+
+on:
+  release:
+    types:
+      - released
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  INTEGRATION: "mysql"
+  ORIGINAL_REPO_NAME: 'newrelic/nri-mysql'
+  TAG: ${{ github.event.release.tag_name }}
+
+jobs:
+
+  publish-to-s3:
+    name: Send release assets to S3
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@main
+        env:
```

Comment:
minor thing but could you modifiy the variables to be consistant with this pr https://github.com/newrelic/nri-haproxy/pull/30/files. ?

Reply:
I have to change nr-haproxy. That PR is "old" and will have to be updated to the names we use here.